test/unit/app/presenters/publishing_api/person_presenter_test.rb

```diff
@@ -119,15 +119,14 @@ def present(...)
     assert_nil presented_item.content.dig(:details, :image)
   end
 
-  test "it raises if a carrierwave path is used for the image path" do
-    person = build(:person)
-
+  test "it logs if image url is a carrierwave temporary path rather than a well formed url" do
+    person = build(:person, id: 1)
     image = build(:featured_image_data)
-    image.expects(:url).twice.with(:s465).returns("/uploads/carrierwave-tmp/1704204355-973510646575090-0009-0387/s465_test-img.jpg")
-    person.image = image
 
-    assert_raises do
-      present(person).content
-    end
+    image.expects(:url).times(3).with(:s465).returns("/uploads/carrierwave-tmp/1704204355-973510646575090-0009-0387/s465_test-img.jpg")
+    logger.expects(:info).with("PersonPresenter: Person of ID#1 has image with url '/uploads/carrierwave-tmp/1704204355-973510646575090-0009-0387/s465_test-img.jpg'").once
+
+    person.image = image
+    present(person).content
```

Comment:
Whilst I applaud your dedication to test coverage, I'm not sure in this case that testing a logging call is sufficiently valuable to justify the performance and maintenance cost of the test (even though it's very minimal)